projects/media-viewer/src/lib/media-viewer/toolbar/search-bar/search-bar.component.html

```diff
@@ -1,8 +1,9 @@
-<div class="findbar doorHanger" [ngClass]="{ hidden: searchBarHidden }">
+<div class="findbar doorHanger" [ngClass]="{ hidden: searchBarHidden.value }">
```

Comment:
should this have | async instead of .value?

Reply:
no because I use it the the component. When it gets the | async it seems to become a boolean

Reply:
no that's fine, but I mean on that particular line, you need the value for it, and that you should be able to get using async

Reply:
I see, I will give it a shot